testutil/client.go

```diff
@@ -59,6 +59,31 @@ var (
 	SockAddrZero string
 	// SockAddrZeroHttp is the address to the HTTP endpoint of the zero used during tests.
 	SockAddrZeroHttp string
+
+	// R_SockAddr is the address to the gRPC endpoint of the alpha4 used during restore tests.
```

Comment:
do we need all of these new variables?

Reply:
Yes, because of 2 HA and 2 non-ha clusters.